torch/csrc/StorageDefs.h

```diff
@@ -1,5 +1,5 @@
 #pragma once
 struct THPStorage {
   PyObject_HEAD
-  THWStorage *cdata;
+  c10::StorageImpl *cdata;
```

Comment:
should this be StorageImpl or intrusive_ptr<StorageImpl> as it is assigned later?

Reply:
`THWStorage` was a direct alias for `StorageImpl`. We could probably change this to `intrusive_ptr` if you'd like, but that will require changing a lot of usage sites. I'm not sure which one is better

Reply:
It's fine to leave it as is